demoapp/build.gradle

```diff
@@ -12,12 +12,12 @@ apply from: "$rootProject.rootDir/build.common.gradle"
 dependencies {
     implementation project(':tw-tkms-starter')
 
+    
     implementation libraries.curatorFramework
-    if (springBootVersion.contains("2.7")) {
```

Comment:
I guess this isn't needed, because demo app is always using boot 2.6 at the moment?

Reply:
This is needed when testing locally, where I would substitute the default `springBootVersion` parameter with target platform.